### PR TITLE
[eas-cli] un-hide project:*

### PR DESCRIPTION
**Pull request by jkhales**
# Checklist

- [x] I've added an entry to [CHANGELOG.md](https://github.com/expo/eas-cli/blob/main/CHANGELOG.md#main) if necessary.
- [x] I've tagged the changelog entry with `[EAS BUILD API]` if it's a part of a breaking change to EAS Build API (only possible when updating `@expo/eas-build-job` package).

# Why

This removes `fullName` from the log messages as well as the graphQL calls in everything *but* the credentials commands.

The credentials are somewhat involved and will be done in a separate PR to keep things easily reviewable.

# How

* remove fullName in message logs
* replace graphQL calls that use fullName with their `byId` analog.
* remove `hidden` from `project:*` commands.

Also cleaned up the `topic` display:

Before:
Screen Shot 2021-07-07 at 2 10 04 PM
After:
Screen Shot 2021-07-07 at 2 10 26 PM
(another option is to just remove the descriptions of these topics as they are pretty redundant)

Q: Are there any other blockers to removing the `EAS_ENABLE_PROJECT_ID` gating?

# Test Plan

Tested commands in demo repo.
* branch:*
* channel:*
* build:*
* secrets:*

### CODE REVIEW

File: packages/eas-cli/package.json [hunk 1]
```diff
@@ -131,15 +131,27 @@
...
+      },
+      "project": {
+        "description": "manage your project"
+      },
       "release": {
         "description": "manage update releases"
       },
+      "secret": {
+        "description": "manage project and account secrets"
+      },
```
Comment (jkhales) [inline packages/eas-cli/package.json:154]: another option is to just remove the descriptions of these topics as they are pretty redundant

File: packages/eas-cli/src/commands/secret/create.ts [hunk 2]
```diff
@@ -121,9 +120,7 @@ export default class EnvironmentSecretCreate extends Command {
       }
 
       Log.withTick(
-        `️Created a new secret ${chalk.bold(name)} on project ${chalk.bold(
-          `@${accountName}/${slug}`
-        )}.`
+        `️Created a new secret ${chalk.bold(name)} on project with id ${chalk.bold(projectId)}.`
```
Comment (brentvatne) [inline packages/eas-cli/src/commands/secret/create.ts:123]: this feels like a meaningful regression in the quality of the output. i don't think anyone will learn their project uuid and so it seems better to not output it, or to print some human readable identifier for the project and the uuid dimmed beside it. we should look at other clis to see how they handle this, nothing comes to mind off of the top of my head 😂

Comment (jkhales) [reply packages/eas-cli/src/commands/secret/create.ts]: good point

**Comment by jkhales** [comment]
reverted the removal of `fullName` logging. 

This PR now just activates `project:*` and updates a graphQL call to rely on `projectId` instead of `fullName`.

File: packages/eas-cli/package.json [hunk 3]
```diff
@@ -131,15 +131,27 @@
...
+        "description": "manage your account"
+      },
       "build": {
         "description": "build app binaries"
       },
       "channel": {
         "description": "manage update channels"
       },
+      "device": {
+        "description": "manage your Apple devices"
```
Comment (brentvatne) [inline packages/eas-cli/package.json:144]:
```suggestion
        "description": "manage your Apple devices for internal distribution"
```

Comment (brentvatne) [reply packages/eas-cli/package.json:144]: seems like useful context

**Review by brentvatne — APPROVED** [review]

**Pull request merged**